src/screen/ConnectionsList/Connection/index.tsx

```diff
@@ -71,17 +80,46 @@ const Connection: React.FC<IConnectionProps> = ({ connection }) => {
       connected={isConnected}
       errored={isConnectionFailed}
     >
-      <button type="button" disabled={isConnected} onClick={handleConnect}>
-        {isConnecting ? (
-          <Loading>
-            <FiLoader />
-          </Loading>
-        ) : (
-          <FiDatabase />
+      <ContextMenuTrigger id="connection_actions_menu">
+        <button type="button" onClick={handleConnect}>
```

Comment:
```<button type="button" disabled={isConnected} onClick={handleConnect}>``` 
`disable` means if you have already got data from `redis` and you trying left click on connection - nothing will happen, but in your case there is new `request` into `redis`, like `refresh` action

dirty solution is `<button type="button" onClick={() => isConnected ? null : handleConnect}>` and it works as expected

[right one](https://github.com/react-component/tooltip/issues/18#issuecomment-411476678), but `cursor` on connection name will be `default`, however we expect `pointer` type

Reply:
Thanks for the feedback!

I wasn't sure how to handle this. I saw there's a check for `isConnected` on `handleConnect`, but it didn't seem to work for the expected `disable` behaviour.

The workaround you mentioned works pretty well. I personally think it is okay to not have `cursor: pointer` when already connected, because there isn't a left click action.